src/components/UpdatePaymentDetailsPopup/UpdatePaymentDetailsPopup.js

```diff
@@ -173,6 +173,7 @@ const UpdatePaymentDetailsPopup = () => {
             step: PAYMENT_DETAILS_STEPS.SUCCESS
           })
         );
+        dispatch(init({ displayGracePeriodError: false }));
```

Comment:
Maybe we can add a reducer for updating `displayGracePeriodError`, because I think that this will look better than calling `init` every time we want to change the value

Reply:
Something simmilar to `updatePaymentMethods` reducer in publisherConfigSlice.js

Reply:
added